dali/base/dadfs.cpp

```diff
@@ -7929,6 +7956,14 @@ IDistributedSuperFile *CDistributedFileDirectory::createSuperFile(const char *_l
 }
 
 // MORE: This should be implemented in DFSAccess later on
+IDistributedSuperFile *CDistributedFileDirectory::createNewSuperFile(ISuperFileDescriptor *desc, IUserDescriptor *user, bool includeports)
```

Comment:
As when creating a IDistributeFile/createNew, shouldn't supply a IUserDescriptor this stage, as it is unattached.
A IUserDescriptor should only be needed when attach()'ing to DFS, at which point permissions are checked.